modules/table/src/main/java/org/apache/ignite/internal/table/distributed/disaster/DisasterRecoveryManager.java

```diff
@@ -575,10 +595,8 @@ private static <T> boolean containsOrEmpty(T item, Collection<T> collection) {
         return collection.isEmpty() || collection.contains(item);
     }
 
-    /**
-     * Converts internal raft node state into public local partition state.
-     */
-    private static LocalPartitionStateEnum convertState(State nodeState) {
+    /** Converts internal raft node state into public local partition state. */
```

Comment:
Why do you keep fixing formatting for comments that you have not changed? Text is the same, literally every letter is the same, I don't get it. We discussed it, like, 10 times already. Is this the impulse that you can't control? In that case I already offered you fixing the entire project, you rejected.

Reply:
I ask you to be more polite, and to be less emotional and more professional.
I said that I would stop formatting the text and making changes that did not affect my changes.
I changed this particular method a little for the task and therefore formatted the documentation, purely for beauty.
I’ll try to answer your question again in advance, no, I won’t make any rules or big changes for the entire project.

Reply:
> purely for beauty.

We discussed this many times, and I thought that we had an agreement. Apparently we don't. And I feel stupid, because I have to repeat the same comment over and over again, without any success.
> I changed this particular method a little for the task and therefore formatted the documentation

Did you change the documentation?
Let me rephrase the question. Let's assume that you changed line number 15 in the code. What's the range of lines in the code that allow for changes with _very_ subjective value, the ones that I would call _useless_? Lines 14-16, or maybe 12-18? How do you decide that?

Reply:
Yes, we discussed this more than once, for my part I feel violence and intolerance for a different view of things.

> Did you change the documentation?

No, I just changed the number of lines.

> What's the range of lines in the code that allow for changes with very subjective value, the ones that I would call useless?

In this question you mention about yourself, I cannot answer it.

> Lines 14-16, or maybe 12-18?

Don't know.

> How do you decide that?

For me, if a multi-line comment can fit into one, then it looks more expressive, but not 100%, it depends on the case and environment.

Reply:
> In this question you mention about yourself, I cannot answer it.

That was a question directed to you, maybe I phrased it poorly. But you already replied that you don't know the answer, so that's fine

> For me, if a multi-line comment can fit into one, then it looks more expressive, but not 100%, it depends on the case and environment.

This is what we discussed. That you won't be doing such changes, because it's all too subjective. It you have a desire to fix formatting of comments in the project - then do it in the entire project. Otherwise I'm against these changes in your PRs, and you know that.